src/Orleans/Streams/PubSub/MemoryPubSub.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Runtime;
+
+namespace Orleans.Streams.PubSub
+{
+    internal class MemoryPubSub : IStreamPubSub
+    {
+        private MemoryPubSubStore store;
```

Comment:
Not sure why this class wraps the store.  Seems like a simple pass-through?  Other pubsub systems are pass-through because they call to their global state (grain, or implicit subscription table).  Unclear why this one should do this though.